lib/miq_environment.rb

```diff
@@ -94,6 +94,7 @@ def self.supports_command?(cmd)
 
       begin
         # If 'which apachectl' returns non-zero, it wasn't found
+        require "awesome_spawn"
         AwesomeSpawn.run!(which, :params => [cmd])
       rescue
```

Comment:
I think part of the problem here is that this is a bare rescue instead of rescuing the more specific`AwesomeSpawn::CommandResultError`.  If this was just the command error, then a different error would be raise (specifically `AwesomeSpawn::NoSuchFileError`)

Reply:
Yeah so how about we also `rescue AwesomeSpawn::CommandResultError` ?

Reply:
I don't think this would raise `NoSuchFileError` because `which` exists, but `which systemctl` would return 0 or 1

Reply:
Yeah `AwesomeSpawn::CommandResultError (which exit code: 1)` is what it would raise